README.md

```diff
@@ -115,7 +148,30 @@ An example of extra specs json would look like this:
+    }
+```
+
+*NOTE*: The `extra_context` spec adds a map of key/value pairs that are expanded in the `runner_install_template`.
```

Comment:
```markdown
that **may** be expected
```

The `extra_context` spec allows you to add arbitrary key/value pairs to your custom template, which enables you to write any custom template you may need.